docs/workstation_release_management.rst

```diff
@@ -88,7 +88,7 @@ Step 5: Build and deploy the packages to ``apt-qa``
    make build-debs
 
 3. Save and publish :doc:`build metadata <build_metadata>`.
-4. Add your packages to a new branch called ``release`` in https://github.com/freedomofpress/securedrop-apt-prod.
+4. Add your packages to a new branch called ``release`` in https://github.com/freedomofpress/securedrop-apt-prod. For Packages with compiled Rust (currently ``securedrop-proxy``), also include the ``-dbgsym`` package.
```

Comment:
It's anything that has compiled code, which is proxy and once https://github.com/freedomofpress/securedrop-client/pull/2178 is merged, also the client, which has a few Python extensions that are compiled.

Reply:
Maybe we can just say something like "Include all packages built by the client, including ``-dbgsym`` ones."?